media/server/ipc/source/SessionManagementServer.cpp

```diff
@@ -25,10 +25,78 @@
 #include "IWebAudioPlayerModuleService.h"
 #include "RialtoServerLogging.h"
 #include <IIpcServerFactory.h>
+#include <grp.h>
+#include <pwd.h>
 #include <sys/stat.h>
+#include <sys/types.h>
+#include <unistd.h>
 
+namespace
+{
+const uid_t kNoOwnerChange = -1; // -1 means chown() won't change the owner
+const gid_t kNoGroupChange = -1; // -1 means chown() won't change the group
+} // namespace
 namespace firebolt::rialto::server::ipc
 {
+static size_t getBufferSizeForPasswordStructureCalls()
```

Comment:
placing functions in empty namespaces instead of static functions is more c++ style :)

Reply:
Thanks, I didn't know this